1_space_odyssey/css/styles.css

```diff
@@ -1,17 +1,34 @@
+
 body {
-    background-color: #000;
-    background-image: url("../img/cover_desktop.jpg");
-    background-position: center;
-    background-size: cover;
-    background-repeat: no-repeat;
-    height: 100vh;
-    transition: fade ease-in;
+  background-color: #000;
+  background-image: url("../img/cover_desktop.jpg");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  transition: fade ease-in;
 }
 
 h1 {
-  font-family: 'Ubuntu', sans-serif;
-  color: #FFF;
-  text-align: center;
-  margin: 50vh auto;
-  font-size: 3em;
+font-family: 'Ubuntu', sans-serif;
```

Comment:
please revert the CSS changes, they are not needed.

Reply:
I didn't touch the CSS file, I only deleted the merge conflict issues